chpl/chpl-service/src/main/java/gov/healthit/chpl/FeatureList.java

```diff
@@ -7,6 +7,7 @@ private FeatureList() {
     public static final String CHANGE_REQUEST = "change-request";
     public static final String CONFORMANCE_METHOD = "conformance-method";
     public static final String ENHANCED_UPLOAD = "enhanced-upload";
+    public static final String EFFECTIVE_RULE_DATE_PLUS_18_MONTHS = "effective-rule-date-plus-18-months";
```

Comment:
Can you move this up a line? It's not alphabetical, which means the PR I'm putting together that has it alphabetized with the other flags will have a merge conflict, or a compilation error, as we're trying to insert the same row in two places